MvvmCross-AndroidSupport/MvvmCross.Droid.Support.Fragment/MvxViewPagerFragmentInfo.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Android.Support.V4.App;
+using MvvmCross.Core.ViewModels;
+
+namespace MvvmCross.Droid.Support.V4
+{
+    public class MvxViewPagerFragmentInfo
+    {
+        public MvxViewPagerFragmentInfo(string title, Type fragmentType, Type viewModelType, object parameterValuesObject = null)
+            : this(title, null, fragmentType, viewModelType, parameterValuesObject)
+        {
+        }
+
+        public MvxViewPagerFragmentInfo(string title, string tag, Type fragmentType, Type viewModelType,
+                            object parameterValuesObject = null)
+        {
+            Title = title;
+            Tag = tag ?? title;
```

Comment:
I think this needs to be processed a bit. Meaning I don't think we are allowed to put anything inside of a tag. But I might be wrong.